src/components/DatePicker/DatePickerComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import DateTimePicker from "@react-native-community/datetimepicker";
+import { DatePickerComponentProps as Props } from "./DatePickerComponentType";
+
+const DatePickerComponent: React.FC<Props> = ({
+  value,
+  onChange,
+  mode,
+  toggleVisibility,
+}) => {
+  return (
+    <DateTimePicker
+      value={value}
+      mode={mode}
+      onChange={(_event: any, data: any) => {
+        toggleVisibility();
+        onChange(null, data);
+      }}
+    />
+  );
+};
+
+export default DatePickerComponent;
```

Comment:
will the right colors show up given the theme? i think thats the last thing that's missing

Reply:
The date picker I added ([material-ui](https://material-ui-pickers.dev/api/DatePicker)) and the old one ([react-native-community/datetimepicker](https://github.com/react-native-community/datetimepicker)) do not have any color customization or styling. So theme will have no affect on them, but that is not a bug I introduced or anything it was how it was and I don't think there is anything we can do about it.

Reply:
this will also need the theme for android (not ios though)